test/Operators/hybrid/cuda.jl

```diff
@@ -91,14 +91,28 @@ end
     # Define hv GPU space
     hv_center_space_gpu, hv_face_space_gpu = hvspace_3D_sphere(gpu_context)
 
-    z = Fields.coordinate_field(hv_face_space_gpu).z
+    coords = Fields.coordinate_field(hv_face_space_gpu)
+    z = coords.z
 
     gradc = Operators.GradientF2C()
 
     @test parent(Geometry.WVector.(gradc.(z))) ≈
           parent(Geometry.WVector.(ones(hv_center_space_gpu)))
-end
 
+
```

Comment:
Can we add a comment what this issue this fixes? (https://github.com/CliMA/ClimaCore.jl/issues/1226)